src/Simulators/schedule-callback-args.js

```diff
@@ -0,0 +1,160 @@
+import { inspect } from 'util';
+
+/**
+ * @typedef {Object} UpdateEvent
+ * @property {Number} elapsed - Milliseconds after the `scheduleCallbackArgs()`
+ *     call to invoke the callback. `0` will invoke immediately. The same
+ *     number twice will invoke twice at the same (rough) time, but in two
+ *     different turns of the event loop.
+ *     Each `elapsed` value represents time elapsed from the initial call,
+ *     not the time between subsequent calls. So if you want to call the
+ *     callback three times, with a 1-second gap before each time, your
+ *     schedule would be:
+ *     [
+ *       { elapsed: 1000, args: args },
+ *       { elapsed: 2000, args: args },
+ *       { elapsed: 3000, args: args }
+ *     ]
+ *
+ * @property {*[]|Function} args - Array of values to use as function
+ *     arguments at the appointed time. Can also be a function that returns
+ *     such an array.
+ */
+
+/**
+ * Turn any value until a function returning that value.
+ * Args props can take arrays, or array-returning lambdas for lazy delivery.
+ * Unify that interface to lambdas early.
+ * @private
+ */
+function lift(value) {
```

Comment:
I don't think I'd really consider this a `lift` - would expect a lift to lift a value into some form of collection. Think we can find a better name that will make more sense to our target contributing audience without going into FP jargon